crates/protocol/src/registry.rs

```diff
@@ -161,6 +161,16 @@ impl PackageId {
     pub fn name(&self) -> &str {
         &self.id[self.colon + 1..]
     }
+
+    /// Check if string is a valid namespace.
+    pub fn is_valid_namespace(namespace: &str) -> bool {
+        const SUPPORTS_NESTED_NAMESPACES: bool = false;
```

Comment:
What's this about?

Reply:
From my understanding, `gh:user:project` nested namespaces will be eventually supported when the tooling catches up. Wanted to make it easy to support but not enable yet. Can remove it or refactor if you have a better convention.

Reply:
I knew the compiler would strip it out at compile-time. Didn't think it merited a build flag. And wanted to have the solution easy to enable in the code.

But I'll push a change to remove it and only validate non-nested namespaces.

Reply:
Removed